source/partials/wp_get_environment_type.md

```diff
@@ -0,0 +1,15 @@
+WordPress 5.5 [introduced the `wp_get_environment_type` function](https://make.wordpress.org/core/2020/07/24/new-wp_get_environment_type-function-in-wordpress-5-5/).
+
+`wp_get_environment_type` allows you to define a constant within `wp-config.php`:
```

Comment:
Users should not define this constant on Pantheon. We will automatically define the correct value in wp-config-pantheon.php based on the environment variables set by the platform.

It is helpful to still have the table below as reference, so users know what values to test for when calling `wp_get_environment_type()`.